static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -165,15 +166,37 @@ export function replaceHeadersWithBold(markdown: string) {
 
 function RootCauseDescription({cause}: {cause: AutofixRootCauseData}) {
   return (
-    <Fragment>
-      <CauseDescription
-        dangerouslySetInnerHTML={{
-          __html: marked(replaceHeadersWithBold(cause.description)),
-        }}
-      />
+    <CauseDescription
+      dangerouslySetInnerHTML={{
+        __html: marked(replaceHeadersWithBold(cause.description)),
+      }}
+    />
+  );
+}
+
+function RootCauseContext({
+  cause,
+  repos,
+}: {
+  cause: AutofixRootCauseData;
+  repos: AutofixRepository[];
+}) {
+  const unitTestFileExtension = cause.unit_test?.file_path
+    ? getFileExtension(cause.unit_test.file_path)
+    : undefined;
+  const unitTestLanguage = unitTestFileExtension
+    ? getPrismLanguage(unitTestFileExtension)
+    : undefined;
+
+  return (
+    <RootCauseContextContainer>
       {cause.reproduction && (
```

Comment:
you have `cause.reproduction` here, but didn't you rename that field in the backend?

Reply:
`reproduction_instructions` is only for the pydantic model sent to the llm to prompt it